src/assets/images/landing/FifthBG.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="1920" height="500" viewBox="0 0 1920 500" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="1920" height="500">
+<rect width="1920" height="500" fill="#171C29"/>
+</mask>
+<g mask="url(#mask0)">
+<rect width="1920" height="500" fill="#171C29"/>
+<g opacity="0.15">
```

Comment:
when naming picture it's better to describe what is it
illustration_monkey for example